interop/tapir/src/test/scala/caliban/interop/tapir/TapirAdapterSpec.scala

```diff
@@ -483,9 +483,7 @@ object TapirAdapterSpec {
       fromMetadata(
         asStringAlways.map(error => Left(HttpError(error, StatusCode.UnprocessableEntity))),
         ConditionalResponseAs(
-          _.contentType.exists(
-            MediaType.unsafeParse(_).matches(ContentTypeRange("multipart", "mixed", ContentTypeRange.Wildcard))
-          ),
+          _.contentType.exists(_.startsWith("multipart/mixed")),
```

Comment:
Is it always first?

Reply:
Do you mean first in the list or at the start of the string?

Reply:
Start of the string. Just wanted to double check `startsWith` is good enough